### DISCUSSION TITLE
/group/*dir/email/ should display the mailman admin passwd for chairs&ADs

### DISCUSSION THREAD

**Discussion by evyncke** [ietf-tools/datatracker#3809] [Ideas]
The DT /group/*dir/email/ page should have a way to display the mailman admin password for all users logged in as chairs or ADs. 

Of course this password should not be displayed by default but a button to either display it or resend it to list-owner@ietf.org would be enough.

-éric

**Comment by rjsparks** [comment]
This should go into the conversation about better mailman integration with the datatracker, queued for a workshop at some point: https://github.com/ietf-tools/tools-workshops (specifically https://github.com/ietf-tools/tools-workshops/blob/main/proposals/mail_archive_interfaces.md)

**Reply by evyncke** [reply]
Perfectly fine for me of course (and BTW, as the list passwords are not stored in the clear, then this would rather be a password reset + email sent to list-owner@ietf.org).
